### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by vadim9999** [GroceriStar/react-print-pdf#164]
1.Headers HeaderShowcase and HeaderRecipes are working. HeaderRecipeFromShowcase.js doesn't export properly from the module. Which one I should use in this repo?
2. When I will enable it what should I do next? Have you tried enabling this header in showcase and recipe-antd?

**Comment by vadim9999** [comment]
@atherdon

**Comment by atherdon** [comment]
in the end - we should have only one header component, that will accept
different options inside of it.
So, instead of having
- showcase project - HeaderShowcase
- antd-recipe project HeaderRecipe
- react-print0df Header

we should have only one component wrapper that will fit well for all of
those cases, so we'll be able to delete all of these old components that
have a similar logic

so your goal to have here only one component:
https://github.com/GroceriStar/antd-showcase-components/tree/master/src/components/Header

**Comment by atherdon** [comment]
so, if i was you - i'll try to figure out the difference inside of code and try to make it for for all cases at the same time.
then i'll push new version at npm
then i'll install that latest version at all 3-4 repositories, that have header component and see if it works

**Comment by vadim9999** [comment]
@atherdon Please review ar header

**Comment by atherdon** [comment]
i'm on it.
Should i review it only at this repo or also at recipe-antd and gs/showcase as well?

**Comment by vadim9999** [comment]
Review it at this repo. Now I will update at other recipe-antd and gs/showcase

**Comment by atherdon** [comment]
code looks ok, but i have fetch issue at selector, can you take a look?
and buzz me when i'll need to check it at 2 other repos as well

>

**Comment by vadim9999** [comment]
Header using component Link from react-router-dom. If you will use header without Router you will get Error

**Comment by vadim9999** [comment]
> code looks ok, but i have fetch issue at selector, can you take a look? 

it's in this repo?

**Comment by atherdon** [comment]
yes, try to start it. maybe i did something wrong.

do you mean i should open not the first page? please elaborate your
previous message about router error

>

**Comment by vadim9999** [comment]
Header will be working with Router. Else you get this Error:
```
Error: Invariant failed: You should not use <Link> outside a <Router>
```
Because in Header we are using component Link that works in component BrowserRouter.
We can fix this replacing component Link by <a href =""></a>

**Comment by atherdon** [comment]
ok, so how i can exactly see a working header?
should i open some sub-page, right?
aka `localhost/prf1` will work?

>

**Comment by vadim9999** [comment]
you can type localhost:3000/ 
In react-print-pdf I should add in BrowserRouter header that it will appear on all paths

**Comment by vadim9999** [comment]
In recipe-ant I also add it to router

**Comment by atherdon** [comment]
should i run react-print-pdf locally again?

>

**Comment by vadim9999** [comment]
yes

**Comment by atherdon** [comment]
​still got an error. maybe i should downgrade fetch? it maybe will help to my local version

![image](https://user-images.githubusercontent.com/1469198/60035465-92580600-96b5-11e9-99d7-e840c72df292.png)

**Comment by vadim9999** [comment]
yes, it's a problem with fetch. I tried to downgrade but it's not a success.
Now you can test showcase.

**Comment by vadim9999** [comment]
Summary 
1. In all 3 projects: react-print-pdf, recipe-antd, showcase are Header that will appear in all paths(/, padf1, groceries).
2. Fetch in this 3 projects doesn't work. Need to fix it.
3. After we will fix fetch we should add a listener on each button that is in Header. Now Header It's just buttons that nothing doing.

**Comment by atherdon** [comment]
ok, got it, then fixing fetch is my goal in order to make our react
frontend work.
i also exploring how to make an advanced travis setup, so at fetch we can
not just build it and run tests, but also can install a clean react app
separately, add fetch there and try to run it in order to see errors like
this

**Comment by vadim9999** [comment]
Do you mean add here travic ci?

**Comment by atherdon** [comment]
it's more like add at travis ci +200% of complexity but for developers it
will work as magic

**Comment by atherdon** [comment]
i'm wroking on it. looks like it will be fixed soon. not ideal but - right now we don't have that eror. updating showcase and recipe-ntd with the same logic
